drupal/settings/all.settings.php

```diff
@@ -9,6 +9,7 @@
  */
 
 // phpcs:disable Drupal.Classes.ClassFileName.NoMatch
+$govcms_settings = __DIR__;
```

Comment:
We only need to set to the current inclusion dir, all services and settings files reside here.

Reply:
Something nags me on this one. That setting will usually be set in Drupal so that a (paas) client can point to any arbitrary location override location. Do we know the implications of overriding it like this? Should we only set it if it's not already set?

Reply:
Actually the value I'm thinking of is `$govcms_includes` see [here](https://github.com/govCMS/govcms8-scaffold-paas/blob/develop/web/sites/default/settings.php#L48). I realise don't know what `$govcms_settings` is.

Reply:
OK, i think somewhere the scaffold called it govcms_includes and the scaffolding tooling started calling it govcms_settings. Drupal doesn't complain when it can't find a services.yml file, which is why it hasn't come up.

So can I suggest we align this variable name.